scripts/functions/support

```diff
@@ -356,6 +356,19 @@ __rvm_setup_sudo_function_Other()
     {
       \command \sudo "$@"
     }
+  elif 
```

Comment:
Instead of modifying `__rvm_setup_sudo_function_Other` you should add new function

```
__rvm_setup_sudo_function_OpenBSD()
{
...
}
```

and implement this logic only there.

Reply:
I thought about that, but doas is available on other platforms (eg, https://www.freebsd.org/cgi/man.cgi?query=doas&sektion=1&manpath=freebsd-release-ports). It seems like if it's there, we should use it.

Reply:
I don't think this is solved the way I suggested?